Peekabook/Peekabook/Presentation/Recommend/VC/RecommendVC.swift

```diff
@@ -18,26 +18,282 @@ final class RecommendVC: UIViewController {
+            make.centerX.equalToSuperview()
+            make.width.equalToSuperview()
+            make.height.equalTo(36)
+        }
+                
+        recommendLabel.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().offset(14)
+        }
+                
+        lineView.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalTo(recommendLabel.snp.trailing).offset(8)
+            make.width.equalTo(1)
+            make.height.equalTo(12)
+        }
+                
+        personName.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalTo(lineView.snp.trailing).offset(8)
+        }
+                
+        recommendView.snp.makeConstraints { make in
+            make.top.equalTo(recommendHeader.snp.bottom).offset(10)
+            make.leading.equalTo(recommendLabel)
+            make.width.equalTo(307)
+            make.height.equalTo(169)
+        }
+                
+        recommendMaxLabel.snp.makeConstraints { make in
+            make.top.equalTo(recommendBox.snp.bottom).offset(8)
+            make.trailing.equalTo(recommendBox.snp.trailing)
+        }
     }
 }
 
 // MARK: - Methods
+
+extension RecommendVC {
+    private func setDelegate() {
+        recommendView.delegate = self
+    }
+    
+    @objc private func popToSearchView() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    // TODO: - push 함수 작성 필요
+    @objc private func presentToPopUpView() {
+        let popupViewController = ConfirmPopUpViewController()
+        popupViewController.modalPresentationStyle = .overFullScreen
+        self.present(popupViewController, animated: false)
+    }
+    
+    private func configButton() {
+        touchBackButton.setImage(ImageLiterals.Icn.back, for: .normal)
+        touchCheckButton.setImage(ImageLiterals.Icn.check, for: .normal)
+    }
+    
+    private func configImageView() {
+        bookImgView.image = ImageLiterals.Sample.book1
+    }
+    
+    private func registerForKeyboardNotification() {
+        NotificationCenter.default.addObserver(self,
+            selector: #selector(keyBoardShow),
+            name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self,
+            selector: #selector(keyboardHide),
+            name: UIResponder.keyboardWillHideNotification, object: nil)
+        }
+
+    private func removeRegisterForKeyboardNotification() {
+        NotificationCenter.default.removeObserver(self,
+            name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self,
+            name: UIResponder.keyboardWillHideNotification, object: nil)
+        }
+    
+    // TODO: - 기종에 따른 테스트 필요
+    @objc private func keyBoardShow(notification: NSNotification) {
```

Comment:
p5
objc 함수들만 따로 빼서 깔끔하게 정리해도 좋을 것 같아요
```suggestion
    // MARK: - @objc Function
    @objc
    private func keyBoardShow(notification: NSNotification) {
    }
```